test/rules/no-fallthrough.js

```diff
@@ -0,0 +1,62 @@
+module.exports = {
+	valid: [
+		"switch(foo) {\n" +
```

Comment:
If airbnb conflicts with the following advice, follow airbnb but let me know:

Function calls and function definitions should be written with no space to the left of the open paren, as in `func(foo)` or `function func(foo)`. But `<keyword> ( <expr> )` as in the above `switch(foo)` should have a space to the left of the open paren.

Reply:
Good point